apps/ycrv/components/list/GaugeList.tsx

```diff
@@ -46,37 +50,43 @@ type TVotesReducer = {
 }
 
 function computeMaxVotesAvailable({state, action}: TVotesReducer): BigNumber {
-	const prevVotes = state.votes[action.gaugeAddress] ?? 0;
+	const prevVotes = state.votes[toAddress(action.gaugeAddress)] ?? 0;
 	return state.maxVotes.sub(state.currentTotal.sub(prevVotes));
 }
 
 function computeNewTotal({state, action}: TVotesReducer): BigNumber {
-	return state.currentTotal.sub(state.votes[action.gaugeAddress] ?? 0).add(action.votes ?? 0);
+	return state.currentTotal.sub(state.votes[toAddress(action.gaugeAddress)] ?? 0).add(action.votes ?? 0);
 }
 
 export function votesReducer(state: TVotesReducerState, action: TVotesReducerAction): TVotesReducerState {
-	const {type, gaugeAddress, votes} = action;
+	const {type, gaugeAddress, maxVotes, votes} = action;
 
 	switch(type) {
+		case 'INIT_MAX': {
+			return {
+				...state,
+				maxVotes: formatBN(maxVotes)
+			};
+		}
```

Comment:
I would suggest we have an INIT action that would calculate the max votes available, instead of the component doing the calculation and dispatching an action with the result. Wdyt?